docs/user-guide/src/bmo/live-iso.md

```diff
@@ -24,6 +24,9 @@ kind: BareMetalHost
 metadata:
   name: live-iso-booted-node
 spec:
+  bmc:
+    address: redfish-virtualmedia://192.168.111.1:8000/redfish/v1/Systems/1
+    credentialsName: live-iso-booted-node-secret
   image:
```

Comment:
Let's add a bootMACAddress to make the example realistic (and avoid confusion on blind copy-paste)? Basically, make it similar to https://book.metal3.io/bmo/provisioning

Reply:
sure!